### PR TITLE
Fix #1076

### LINKED ISSUE #1076: Undefined offset: 1 (enabled HTTP/2 support in the Curl library used by PHP)
https://github.com/facebook/php-graph-sdk/blob/3138249d85e8dc040f15bedd3581a3c616f49005/src/Facebook/Http/GraphRawResponse.php#L107

From today I am getting error like in title. I can see in logs this call:

`Facebook\Http\GraphRawResponse->setHttpResponseCodeFromHeader('HTTP/2 200 ')`

It doesn't match to regex in code: 
`|HTTP/\d\.\d\s+(\d+)\s+.*|`

The problem appeared after we enabled HTTP/2 support in the Curl library used by PHP.

It should be solve problem i guess:
`/HTTP\/\d(?:\.\d)?\s+(\d+)\s+/`

Someone else getting the same error?

We are using SDK v. 5.6.3.

### PR DESCRIPTION
**Pull request by bytestream**
closes #1076

### CODE REVIEW

**Comment by yguedidi** [comment]
@bytestream thanks for this contribution! It looks like the branch is not up to date and that ou have to sign the CLA. Do you plan to do so? @jestinas also opened a fix for the bug, but you opened it first so either you update your PR and I accept yours, or let @jestinas update his PR. :)

**Comment by bytestream** [comment]
@yguedidi signed and updated.

If you could tag a 5.6 release that would be great.

**Comment by yguedidi** [comment]
Thanks @bytestream! Can you please also update the changelog and add a test for this case so it doesn't break later? thanks again

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Version 5 of the Facebook PHP SDK is a complete refactor of version 4. It comes
 - 5.7.0 (2018-00-00)
   - Add `joined` to list of fields to be cast to `\DateTime` (#950)
   - Add `GraphPage::getFanCount()` to get the number of people who like the page (#815)
+  - Fixed HTTP/2 support (#1076)
```
Comment (yguedidi) [inline CHANGELOG.md]: we usually link the PR instead of the issue

File: tests/Http/GraphRawResponseTest.php [hunk 2]
```diff
@@ -90,4 +90,21 @@ public function testCanTransformJsonHeaderValues()
 
         $this->assertEquals($this->jsonFakeHeaderAsArray['x-fb-ads-insights-throttle'], $headers['x-fb-ads-insights-throttle']);
     }
+    
+    public function testHttpResponseCode()
+    {
+        // HTTP/1.0
+        $headers = str_replace("HTTP/1.1", "HTTP/1.0", $this->fakeRawHeader);
```
Comment (yguedidi) [inline tests/Http/GraphRawResponseTest.php]: we use single quotes when possible. Same below

**Review by yguedidi — CHANGES_REQUESTED** [review]
final changes! please also squash your PR in one commit, then I'll merge it, thanks!

**Comment by bytestream** [comment]
Squashed 👍

**Review by yguedidi — APPROVED** [review]

**Comment by yguedidi** [comment]
Thanks @bytestream! will be release soon, once the 5.7.0 milestone is completed!

**Comment by julienbourdeau** [comment]
Any ETA for the 5.7 release? 🙏  The [milestone](https://github.com/facebook/php-graph-sdk/milestone/11) seems complete.

**Comment by julienbourdeau** [comment]
Until this is released, I had to use my fork to use the current branch, I'll get rid of this when 5.7 is out.

If you're interested:

1. fork repository
2. update `composer.json`
```json
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/julienbourdeau/php-graph-sdk"
        }
    ],
    "require": {
        "php": "^7.1.3",
        "facebook/graph-sdk": "5.x-dev"
    },
```
3. delete vendor folder (just in case) `rm -rf vendor/facebook`
4. Update with `composer update`

**Comment by yguedidi** [comment]
@julienbourdeau 5.7.0 finally released! Sorry for the delay!

**Comment by julienbourdeau** [comment]
@yguedidi Great! Thanks, I didn't expect it before christmas. 💯

**Pull request merged**